designs/2022-community-eslint-org/README.md

```diff
@@ -0,0 +1,263 @@
+- Repo: No ESLint repos, only the ones mentioned below (owned by private
+  persons)
+- Start Date: 2022-05-31
+- RFC PR:
+- Authors: [@aladdin-add](https://github.com/aladdin-add),
+  [@MichaelDeBoey](https://github.com/MichaelDeBoey),
+  [@ota-meshi](https://github.com/ota-meshi) &
+  [@voxpelli](https://github.com/voxpelli)
+
+# `@eslint-community` GitHub organisation
+
+## Summary
+
+<!-- One-paragraph explanation of the feature. -->
+
+Facilitate a GitHub organisation
+([@eslint-community](https://github.com/eslint-community)) where community
+members could maintain widely used ESLint related packages that would otherwise
+get abandoned.
```

Comment:
My hope is to have a centralized home for important/widely used eslint-related packages, not just for unmaintained ones. We can outline:
1. Who's the org maintainers? the abilities and responsibilities?
2. Acceptance criteria?
3. Process for Adding new projects? what info should be provided (repo, package name, maintainer, reason)? how/who to evaluate?
4. the package maintainer's abilities and responsibilities?
5. security & funding....

Reply:
> it risks signaling that modules there are better/preferred over plugins outside of it.

But we can document the fact: it's not an official org - just like its name - it's community maintained.

> It also risks adding lots of unrelated maintainers into a shared group with little collective ownership, making it hard to push for an active and progressive maintenance.

The package maintainers have just control of their repo, and no other repos. I think the org admin/eslint team won't take in the regular maintenance. does it make sense to you?

Reply:
> The package maintainers have just control of their repo, and no other repos. I think the org admin/eslint team won't take in the regular maintenance. does it make sense to you?

Maintaining all of these individually rather than collectively with a single streamlined process, that seems unnecessary.

I rather pull them into a place where they can be maintained collectively by a single group, where the processes for each can be shared by the others, thus lowering the burden + lowering the reliance on any individual member of that group.

- [@mysticatea/eslint-plugin](https://github.com/mysticatea/eslint-plugin)
- [eslint-plugin-eslint-comments](https://github.com/mysticatea/eslint-plugin-eslint-comments)
- [eslint-plugin-es](https://github.com/mysticatea/eslint-plugin-es)
- [eslint-plugin-node](https://github.com/mysticatea/eslint-plugin-node)
- [eslint-utils](https://github.com/mysticatea/eslint-utils)
- [eslint-plugin-promise](https://github.com/xjamundx/eslint-plugin-promise)

Reply:
@aladdin-add We _could_ streamline things, but that it's not a goal on its own.

The main goal still remains to maintain abandoned ESLint related packages that are widely depended upon imo.